### PR TITLE
Add packer

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments
Add packer to easy_infra.

## Pull Request Checklist
Thank you for submitting a contribution to `easy_infra`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:
 - [x] Ensure that `make build` completes successfully
 - [X] Rebase your branch against the latest commit of the target branch, which likely should be `master`
 - [X] If there is an issue associated with your Pull Request, align your PR title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -41,8 +41,12 @@ RUN apt-get update && \
 
 # binary downloads
 ARG TFSEC_VERSION="v0.24.1"
+ARG PACKER_VERSION="v1.6.1"
 RUN curl -L https://github.com/liamg/tfsec/releases/download/${TFSEC_VERSION}/tfsec-linux-amd64 -o /usr/local/bin/tfsec && \
```
Comment (ericSeiso) [inline Dockerfile:45]: Tried going to this URL. Got 404. Is this the updated path? https://github.com/liamg/tfsec/releases/tag/v0.24.1/

Comment (JonZeolla) [reply Dockerfile:45]: Touched base with @ericSeiso, link is good as-is.  Resolving

File: Dockerfile [hunk 2]
```diff
@@ -41,8 +41,12 @@ RUN apt-get update && \
...
-    chmod 0755 /usr/local/bin/tfsec
+    chmod 0755 /usr/local/bin/tfsec && \
+    curl -L https://releases.hashicorp.com/packer/${PACKER_VERSION#v}/packer_${PACKER_VERSION#v}_linux_amd64.zip -o /usr/local/bin/packer.zip && \
```
Comment (pb-seiso) [inline Dockerfile:47]: I don't like injecting into URLs like this as it seems brittle, but they don't seem to have `apt` support.

Comment (JonZeolla) [reply Dockerfile:47]: I agree, they have publicly stated they do not currently have a goal to support packaged distribution

**Pull request merged**